ansible/idr-playbooks/group_vars/omero-hosts.yml

```diff
@@ -3,13 +3,14 @@ omero_dbname: idr
 omero_dbpassword: omero
 omero_datadir: /data/OMERO
 
-# TODO: change this to a lower system uid
-omero_system_uid: "{{ idr_secret_omero_system_uid | default(546) }}"
+omero_system_uid: 546
 
 omero_web_runtime_redis: True
 # This refers to a custom IDR release URL, not an official OMERO version
-omero_release: "0.0.7-rc1"
+omero_release: "0.0.7-rc2"
```

Comment:
It'll be in a later commit/PR https://github.com/manics/infrastructure/commit/4668d16b3158fcc80fde9209fb9729689b1b18f8